src/html/settings.html

```diff
@@ -4,346 +4,399 @@
-                <div class="subsettings-details">
-                    On weekdays from <input type="time" id="nag-nanny-from"> to <input type="time" id="nag-nanny-to">&nbsp;, after <input type="number" id="nag-nanny-interval" min="1"> minutes since the last time entry was stopped.
-                </div>
-              </div>
-              <div>
-                <input type="checkbox" id="nag-nanny">
-              </div>
+            <span>Integrations</span>
+          </li>
+          <li class="nav-link" data-tab="telemetry">
+            <div class="nav-icon">
+              <svg width="16" height="16" viewBox="0 0 24 24">
+                <path fill="#fff" d="M6 0c-6 12 4 22 16 16L6 0zM5 24H2l7-6 6 6h-3l-3-2-4 2zM22 2a1 1 0 01-2 1l-3 5-3-3 5-3V1a2 2 0 013 1z"/>
+              </svg>
             </div>
-
-            <div class="field">
-              <div class="field__label">
-                <label for="discard-notification">Idle detection</label>
-                <div class="description">Receive a notification when you've been idle and the timer was running, allowing you to discard the time.</div>
-              </div>
-              <div>
-                <input type="checkbox" id="idle-detection">
-              </div>
+            <span>Telemetry</span>
+          </li>
+          <li class="nav-link" data-tab="account">
+            <div class="nav-icon">
+              <svg width="16" height="16" viewBox="0 0 12 12">
+                <path fill="#fff" d="M10 9C9 9 7 8 8 7c2-5 0-7-2-7-3 0-4 2-2 7 1 1-1 2-2 2-2 0-2 1-2 3h12c0-2 0-3-2-3zm0 0"/>
+              </svg>
             </div>
-
-            <div class="field stop-at-day-end">
-              <div class="field__label">
-                <label for="stop-at-day-end">Stop timer automatically at the end of the day</label>
-                <div class="description">The timer will be stopped automatically when your day ends</div>
-                <div class="subsettings-detail">
-                  <div class="subsettings-details">
-                    <label for="day-end-time">End of day time:</label>
-                    <input type="time" id="day-end-time">
-                  </div>
-                </div>
-              </div>
-
-              <div>
-                <input type="checkbox" id="stop-at-day-end">
-              </div>
-            </div>
-
-            <hr />
-
-            <div class="field">
-              <div class="field__label">
-                <label for="show_post_start_popup">Show post-start popup</label>
-                <div class="description">Opens the edit form immediately after clicking a "Start timer" link</div>
-              </div>
-              <div>
-                <input type="checkbox" id="show_post_start_popup">
-              </div>
+            <span>My Account</span>
+          </li>
+        </ul>
+      </div>
+      <div class="section">
+        <h3>Button Help</h3>
```

Comment:
Same here `Help` should do the job

Reply:
Same as https://github.com/toggl/toggl-button/pull/1595/files#r350984711